README.md

```diff
@@ -20,51 +20,64 @@ Useful links
 Quick Start Guide
 --------------
 
-1. Install Rust:
```

Comment:
Can we have section on debugging or at least run some of the CI tests/regression tests or unit tests?

Reply:
Created https://fortanix.atlassian.net/browse/SALM-565 to address debugging in the future